src/vm/ceeload.h

```diff
@@ -1634,6 +1634,11 @@ class Module
     BOOL                            m_nativeImageProfiling;
     CORCOMPILE_METHOD_PROFILE_LIST *m_methodProfileList;
 
+#if PROFILING_SUPPORTED 
+    DWORD                   m_dwTypeCount;
```

Comment:
I forgot before, you actually want PROFILING_SUPPORTED_DATA instead of PROFILING_SUPPORTED. The issue is that the DAC build does not define PROFILING_SUPPORTED, but if these fields are missing then its layout computations for Module will be wrong. You won't see a bug right now, but if we added new fields to the module after these and used them in the debugger you would see it.

Reply:
Thanks!